### PR TITLE
Rename to Kadai

### PR DESCRIPTION
**Pull request by jamesrdi**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [ ] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I put the ticket in review

### Verified by the reviewer:
- [ ] Commit message format → ADPT-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,116 +1,123 @@
...
 
+The Kadai Adapter repository consists of the kadai adapter, sample connectors to camunda and kadai as well as
+an outbox REST-service and it's SpringBoot-Starter and listeners for camunda. In addition to that there are various
+example and test modules
 
+The sample implementation of the camunda-system-connector uses the outbox-pattern.
+The general concept of this pattern can be found here:
+<a href="https://microservices.io/patterns/data/transactional-outbox.html"> Transactional-Outbox-Pattern</a>.
+How the Camunda BPM Connector uses this pattern can be found
+here: <a href="https://taskana.atlassian.net/wiki/spaces/TAS/pages/881164289/Camunda+BPM+Connector"> Camunda BPM
```
Comment (CRoberto1926) [inline README.md:19]: This is still a taskana link, what should we do here?

File: README.md [hunk 2]
```diff
@@ -1,116 +1,123 @@
...
+1. Variables \
+   When the adapter finds a referenced task for which a kadai task must be started, it checks the variables of the
+   referenced task's process. If they are not already present due to retrieval from the outbox it will attempt to
+   retrieve them from the referenced task's process.
+   These variables are stored in the **custom attributes** of the corresponding kadai task in a HashMap with key *
+   *referenced_task_variables** and value of type String that contains the Json representation of the variables.
+2. Workbaskets \
+   The Adapter does not perform routing of tasks to workbaskets but instead relies on a SPI. A detailed description can
+   be found here:
+   <a href="https://taskana.atlassian.net/wiki/spaces/TAS/pages/995524621/TaskRouting+Service+Provider+Interface">
```
Comment (CRoberto1926) [inline README.md:117]: This is a taskana link, what should we do here?

File: README.md [hunk 3]
```diff
@@ -1,116 +1,123 @@
...
-2.  Workbaskets \
-    The Adapter does not perform routing of tasks to workbaskets but instead relies on a SPI. A detailed description can be found here:
-    <a href="https://taskana.atlassian.net/wiki/spaces/TAS/pages/995524621/TaskRouting+Service+Provider+Interface">TaskRouting Service Provider Interface</a>
-    
-        
 ## Properties
 
-A detailed overview of the adapter properties can be found here: <a href="https://taskana.atlassian.net/wiki/spaces/TAS/pages/996966415/Adapter+Properties"> Adapter Properties</a>
+A detailed overview of the adapter properties can be found
+here: <a href="https://taskana.atlassian.net/wiki/spaces/TAS/pages/996966415/Adapter+Properties"> Adapter Properties</a>
```
Comment (CRoberto1926) [inline README.md:123]: This is a taskana link, what should we do here?

File: camunda-outbox-example-boot/src/test/java/io/kadai/camunda/spring/boot/ArchitectureTest.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package pro.taskana;
+package io.kadai.camunda.spring.boot;
```
Comment (CRoberto1926) [inline camunda-outbox-example-boot/src/test/java/io/kadai/camunda/spring/boot/ArchitectureTest.java:1]: Is this correct? Or should it be in the package io.kadai?

File: ci/mvnsettings.xml [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
-			<id>ossrh</id>
-      <username>44pdzWD8</username>
-      <password>eJZ25rsRdH7ZiePJO52JUy7LyVkNIgXxXEBmnTNH3aOM</password>
-		</server>
-	</servers>
+    <servers>
+        <server>
+            <id>ossrh</id>
+            <username>44pdzWD8</username>
+            <password>eJZ25rsRdH7ZiePJO52JUy7LyVkNIgXxXEBmnTNH3aOM</password>
```
Comment (CRoberto1926) [inline ci/mvnsettings.xml:20]: Can we manage these through some secrets?

File: kadai-adapter-camunda-listener-example/src/main/java/io/kadai/impl/WorkbasketKeySetter.java [hunk 6]
```diff
@@ -50,7 +50,7 @@ private String getDomainName(DelegateExecution execution) {
 
     } catch (Exception e) {
       LOGGER.warn(
-          "Caught while trying to retrieve the taskana.domain property from a process model", e);
+          "Caught {} while trying to retrieve the kadai.domain property from a process model", e);
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener-example/src/main/java/io/kadai/impl/WorkbasketKeySetter.java:53]: I'm not sure that with this change we see the stacktrace anymore. Did you test it?

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_execution_process.bpmn [hunk 7]
```diff
@@ -60,7 +60,7 @@
     </bpmn:endEvent>
     <bpmn:sequenceFlow id="SequenceFlow_1ejf70l" sourceRef="ExclusiveGateway_04oie5i" targetRef="EndEvent_01bvfnt" />
     <bpmn:sequenceFlow id="SequenceFlow_0cw5mod" sourceRef="Task_0c2v1r8" targetRef="ExclusiveGateway_1clkhnx" />
-    <bpmn:serviceTask id="Task_0c2v1r8" camunda:class="pro.taskana.impl.WorkbasketKeySetter">
+    <bpmn:serviceTask id="Task_0c2v1r8" camunda:class="impl.io.kadai.WorkbasketKeySetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_execution_process.bpmn:63]: io.kadai.impl.WorkbasketKeySetter ?

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn [hunk 8]
```diff
@@ -3,23 +3,23 @@
...
+        <camunda:property name="kadai-attributes" value="attribute1,attribute2,attribute3" />
+        <camunda:property name="kadai.domain" value="DOMAIN_A" />
       </camunda:properties>
     </bpmn:extensionElements>
     <bpmn:startEvent id="StartEvent_0c8j229">
       <bpmn:outgoing>SequenceFlow_182avfz</bpmn:outgoing>
     </bpmn:startEvent>
     <bpmn:sequenceFlow id="SequenceFlow_182avfz" sourceRef="StartEvent_0c8j229" targetRef="Task_0k9mjh4" />
-    <bpmn:serviceTask id="Task_0k9mjh4" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0k9mjh4" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn:14]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn [hunk 9]
```diff
@@ -13,13 +13,13 @@
...
         <camunda:properties>
-          <camunda:property name="taskana.classification-key" value="L1050" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_0ncfypl</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_0ptcaku</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:22]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn [hunk 10]
```diff
@@ -31,13 +31,13 @@
...
         <camunda:properties>
-          <camunda:property name="taskana.classification-key" value="L1050" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_17f586o</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_0oqq965</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0si9wqw" name="ComplexVariableRetriever" camunda:class="pro.taskana.impl.ComplexProcessVariableRetriever">
+    <bpmn:serviceTask id="Task_0si9wqw" name="ComplexVariableRetriever" camunda:class="impl.io.kadai.ComplexProcessVariableRetriever">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:40]: io.kadai.impl.ComplexProcessVariableRetriever ?

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java [hunk 11]
```diff
@@ -1,34 +1,34 @@
...
-      "taskana.adapter.outbox.datasource.url";
-  private static final String TASKANA_ADAPTER_OUTBOX_DATASOURCE_USERNAME =
-      "taskana.adapter.outbox.datasource.username";
-  private static final String TASKANA_ADAPTER_OUTBOX_DATASOURCE_PASSWORD =
-      "taskana.adapter.outbox.datasource.password";
-  private static final String TASKANA_ADAPTER_OUTBOX_INITIAL_NUMBER_OF_TASK_CREATION_RETRIES =
-      "taskana.adapter.outbox.initial.number.of.task.creation.retries";
-  private static final String OUTBOX_SYSTEM_PROPERTY = "taskana.outbox.properties";
-  private static final String OUTBOX_SCHEMA_DEFAULT = "taskana_tables";
+  private static final String KADAI_ADAPTER_CREATE_OUTBOX_SCHEMA =
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java:12]: Do we need a migration script for this schema change?

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java [hunk 12]
```diff
@@ -1,34 +1,34 @@
...
+  private static final String KADAI_ADAPTER_OUTBOX_DATASOURCE_URL =
+      "kadai.adapter.outbox.datasource.url";
+  private static final String KADAI_ADAPTER_OUTBOX_DATASOURCE_USERNAME =
+      "kadai.adapter.outbox.datasource.username";
+  private static final String KADAI_ADAPTER_OUTBOX_DATASOURCE_PASSWORD =
+      "kadai.adapter.outbox.datasource.password";
+  private static final String KADAI_ADAPTER_OUTBOX_INITIAL_NUMBER_OF_TASK_CREATION_RETRIES =
+      "kadai.adapter.outbox.initial.number.of.task.creation.retries";
+  private static final String OUTBOX_SYSTEM_PROPERTY = "kadai.outbox.properties";
+  private static final String OUTBOX_SCHEMA_DEFAULT = "kadai_tables";
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java:29]: Do we need a migration script for this schema change?

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java [hunk 13]
```diff
@@ -17,17 +17,17 @@
...
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(TaskanaOutboxSchemaCreator.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(KadaiOutboxSchemaCreator.class);
   private static final String POSTGRESQL = "PostgreSQL";
   private static final String H2 = "H2";
   private static final String SQL = "/sql";
-  private static final String DB_SCHEMA = SQL + "/h2/taskana-outbox-schema.sql";
-  private static final String DB_SCHEMA_DB2 = SQL + "/db2/taskana-outbox-schema-db2.sql";
+  private static final String DB_SCHEMA = SQL + "/h2/kadai-outbox-schema.sql";
+  private static final String DB_SCHEMA_DB2 = SQL + "/db2/kadai-outbox-schema-db2.sql";
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java:27]: Do we need a migration script for this schema change?

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java [hunk 14]
```diff
@@ -17,17 +17,17 @@
...
   private static final String DB_SCHEMA_POSTGRES =
-      SQL + "/postgres/taskana-outbox-schema-postgres.sql";
-  private static final String DB_SCHEMA_ORACLE = SQL + "/oracle/taskana-outbox-schema-oracle.sql";
+      SQL + "/postgres/kadai-outbox-schema-postgres.sql";
+  private static final String DB_SCHEMA_ORACLE = SQL + "/oracle/kadai-outbox-schema-oracle.sql";
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java:30]: Do we need a migration script for this schema change?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_execution_process.bpmn [hunk 15]
```diff
@@ -60,7 +60,7 @@
     </bpmn:endEvent>
     <bpmn:sequenceFlow id="SequenceFlow_1ejf70l" sourceRef="ExclusiveGateway_04oie5i" targetRef="EndEvent_01bvfnt" />
     <bpmn:sequenceFlow id="SequenceFlow_0cw5mod" sourceRef="Task_0c2v1r8" targetRef="ExclusiveGateway_1clkhnx" />
-    <bpmn:serviceTask id="Task_0c2v1r8" camunda:class="pro.taskana.impl.WorkbasketKeySetter">
+    <bpmn:serviceTask id="Task_0c2v1r8" camunda:class="impl.io.kadai.WorkbasketKeySetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_execution_process.bpmn:63]: io.kadai.impl.WorkbasketKeySetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn [hunk 16]
```diff
@@ -3,23 +3,23 @@
...
+        <camunda:property name="kadai-attributes" value="attribute1,attribute2,attribute3" />
+        <camunda:property name="kadai.domain" value="DOMAIN_A" />
       </camunda:properties>
     </bpmn:extensionElements>
     <bpmn:startEvent id="StartEvent_0c8j229">
       <bpmn:outgoing>SequenceFlow_182avfz</bpmn:outgoing>
     </bpmn:startEvent>
     <bpmn:sequenceFlow id="SequenceFlow_182avfz" sourceRef="StartEvent_0c8j229" targetRef="Task_0k9mjh4" />
-    <bpmn:serviceTask id="Task_0k9mjh4" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0k9mjh4" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn:14]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_big_complex_variables.bpmn [hunk 17]
```diff
@@ -13,13 +13,13 @@
...
         <camunda:properties>
-          <camunda:property name="taskana.classification-key" value="L1050" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_0ncfypl</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_0ptcaku</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0sh7pjm" name="BigComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="BigComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_big_complex_variables.bpmn:22]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn [hunk 18]
```diff
@@ -13,13 +13,13 @@
...
         <camunda:properties>
-          <camunda:property name="taskana.classification-key" value="L1050" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_0ncfypl</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_0ptcaku</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:22]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn [hunk 19]
```diff
@@ -31,13 +31,13 @@
...
         <camunda:properties>
-          <camunda:property name="taskana.classification-key" value="L1050" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_17f586o</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_0oqq965</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0si9wqw" name="ComplexVariableRetriever" camunda:class="pro.taskana.impl.ComplexProcessVariableRetriever">
+    <bpmn:serviceTask id="Task_0si9wqw" name="ComplexVariableRetriever" camunda:class="impl.io.kadai.ComplexProcessVariableRetriever">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:40]: io.kadai.impl.ComplexProcessVariableRetriever ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_different_domains.bpmn [hunk 20]
```diff
@@ -73,7 +73,7 @@
     <bpmn:sequenceFlow id="Flow_0jkseco" sourceRef="UserTask_1u5oqhv" targetRef="Gateway_0mqata4" />
     <bpmn:sequenceFlow id="Flow_07abxz0" sourceRef="UserTask_1kfz8x" targetRef="Gateway_0mqata4" />
     <bpmn:sequenceFlow id="Flow_0h2nc8t" sourceRef="Gateway_0mqata4" targetRef="EndEvent_0jgafnj" />
-    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_different_domains.bpmn:76]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_empty_extension_property.bpmn [hunk 21]
```diff
@@ -24,7 +24,7 @@
       <bpmn:incoming>SequenceFlow_02trahh</bpmn:incoming>
     </bpmn:endEvent>
     <bpmn:sequenceFlow id="SequenceFlow_02trahh" sourceRef="UserTask_1adz61s" targetRef="EndEvent_1rp66xw" />
-    <bpmn:serviceTask id="ServiceTask_13eimrx" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="ServiceTask_13eimrx" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_empty_extension_property.bpmn:27]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_incorrect_workbasket_key.bpmn [hunk 22]
```diff
@@ -4,7 +4,7 @@
     <bpmn:startEvent id="StartEvent_1">
       <bpmn:outgoing>SequenceFlow_1eiligh</bpmn:outgoing>
     </bpmn:startEvent>
-    <bpmn:serviceTask id="ServiceTask_0vg4xui" name="TaskWithIncorrectWorkbasketKey" camunda:class="pro.taskana.impl.InvalidWorkbasketKeySetter">
+    <bpmn:serviceTask id="ServiceTask_0vg4xui" name="TaskWithIncorrectWorkbasketKey" camunda:class="impl.io.kadai.InvalidWorkbasketKeySetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_incorrect_workbasket_key.bpmn:7]: io.kadai.impl.InvalidWorkbasketKeySetter ?

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_multiple_tasks_and_complex_variables.bpmn [hunk 23]
```diff
@@ -13,14 +13,14 @@
...
-          <camunda:property name="taskana-attributes" value="attribute1" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
+          <camunda:property name="kadai-attributes" value="attribute1" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>Flow_0u2iveq</bpmn:incoming>
       <bpmn:outgoing>Flow_07abxz0</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_multiple_tasks_and_complex_variables.bpmn:23]: io.kadai.impl.ComplexProcessVariableSetter ?

File: kadai-adapter-camunda-wildfly-example/src/main/java/io/kadai/adapter/KadaiAdapterWildFlyApplication.java [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+@EnableScheduling
+@ComponentScan(
+    basePackages = {
+      "io.kadai.adapter",
+      "io.kadai.adapter.configuration",
+      "io.kadai",
+      "io.kadai.adapter.systemconnector.camunda.config",
+      "io.kadai.adapter.kadaiconnector.config"
+    })
+@SuppressWarnings("checkstyle:Indentation")
```
Comment (CRoberto1926) [inline kadai-adapter-camunda-wildfly-example/src/main/java/io/kadai/adapter/KadaiAdapterWildFlyApplication.java:26]: But why? Can't we just format the file properly?

File: qa/hooks/commit-msg [hunk 25]
```diff
@@ -4,7 +4,7 @@ COMMIT_MESSAGE="$(cat $MSG_FILE)"
 REGEX='^((ADPT)-[0-9]+):'
 if [[ $COMMIT_MESSAGE =~ $REGEX ]]; then
  TICKET=${BASH_REMATCH[1]}
- RESULT=$(curl -s https://taskana.atlassian.net/rest/api/3/issue/$TICKET)
+ RESULT=$(curl -s https://kadai.atlassian.net/rest/api/3/issue/$TICKET)
```
Comment (CRoberto1926) [inline qa/hooks/commit-msg:7]: is this URL correct?

File: kadai-adapter-camunda-listener-example/src/main/java/io/kadai/impl/WorkbasketKeySetter.java (hunk 6 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener-example/src/main/java/io/kadai/impl/WorkbasketKeySetter.java:53]: Yes you're right, we still don't see the stacktrace, I will change it

File: README.md (hunk 1 above)
Comment (jamesrdi) [reply README.md:19]: I will erase it for the moment

File: README.md (hunk 2 above)
Comment (jamesrdi) [reply README.md:117]: I will erase it for the moment

File: README.md (hunk 3 above)
Comment (jamesrdi) [reply README.md:123]: I will erase it for the moment

File: camunda-outbox-example-boot/src/test/java/io/kadai/camunda/spring/boot/ArchitectureTest.java (hunk 4 above)
Comment (jamesrdi) [reply camunda-outbox-example-boot/src/test/java/io/kadai/camunda/spring/boot/ArchitectureTest.java:1]: You're right, it should be in io.kadai

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_execution_process.bpmn (hunk 7 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_execution_process.bpmn:63]: You're right, I will change it

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn (hunk 8 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn:14]: You're right, I will change it

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn (hunk 9 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:22]: You're right, I will change it

File: kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn (hunk 10 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener-example/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:40]: You're right, I will change it

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java (hunk 11 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java:12]: No we don't need migration script

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java (hunk 12 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/CamundaListenerConfiguration.java:29]: No we don't need migration script

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java (hunk 13 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java:27]: No we don't need migration script

File: kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java (hunk 14 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java:30]: No we don't need migration script

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_execution_process.bpmn (hunk 15 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_execution_process.bpmn:63]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn (hunk 16 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn:14]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_big_complex_variables.bpmn (hunk 17 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_big_complex_variables.bpmn:22]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn (hunk 18 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:22]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn (hunk 19 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn:40]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_different_domains.bpmn (hunk 20 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_different_domains.bpmn:76]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_empty_extension_property.bpmn (hunk 21 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_empty_extension_property.bpmn:27]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_incorrect_workbasket_key.bpmn (hunk 22 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_incorrect_workbasket_key.bpmn:7]: You're right, I will change it

File: kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_multiple_tasks_and_complex_variables.bpmn (hunk 23 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_multiple_tasks_and_complex_variables.bpmn:23]: You're right, I will change it

File: kadai-adapter-camunda-wildfly-example/src/main/java/io/kadai/adapter/KadaiAdapterWildFlyApplication.java (hunk 24 above)
Comment (jamesrdi) [reply kadai-adapter-camunda-wildfly-example/src/main/java/io/kadai/adapter/KadaiAdapterWildFlyApplication.java:26]: Let's just leave it like that for the moment

File: qa/hooks/commit-msg (hunk 25 above)
Comment (jamesrdi) [reply qa/hooks/commit-msg:7]: This URL is not used anymore, I will delete it

File: camunda-outbox-example-boot/src/test/java/io/kadai/camunda/spring/boot/ArchitectureTest.java (hunk 4 above)
Comment (jamesrdi) [reply camunda-outbox-example-boot/src/test/java/io/kadai/camunda/spring/boot/ArchitectureTest.java:1]: It is correct

File: ci/mvnsettings.xml (hunk 5 above)
Comment (jamesrdi) [reply ci/mvnsettings.xml:20]: We can do it in the future, this PR is only for renaming relevant

**Pull request closed without merging**